stories/Content/Tag/DropdownTag/DropdownTag.stories.js

```diff
@@ -1,10 +1,10 @@
 import React from "react";
 
-import { ICON_TYPES, TAG_COLORS } from "../../confetti-ds/src/constants";
-import { DropdownTag as Component  } from "../../confetti-ds/src";
+import { ICON_TYPES, TAG_COLORS } from "../../../../confetti-ds/src/constants";
+import { DropdownTag as Component  } from "../../../../confetti-ds/src";
 
 export default {
-  title: "Tag/Dropdown Tag",
+  title: "Content/Tag/Dropdown Tag/Dropdown Tag",
```

Comment:
```suggestion
  title: "Content/Tag/Dropdown Tag",
```

Dropdown Tag stories from Tag QA should use this title too.

Reply:
fixed